sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -404,6 +404,13 @@ private[spark] object HiveUtils extends Logging {
     propMap.put(ConfVars.METASTORE_EVENT_LISTENERS.varname, "")
     propMap.put(ConfVars.METASTORE_END_FUNCTION_LISTENERS.varname, "")
 
+    // Copy any "spark.hadoop.foo=bar" system properties into conf as "foo=bar"
```

Comment:
@yaooqinn Please follow what @tejasapatil said and create a util function. In addition, `newTemporaryConfiguration` is being used for `SparkSQLCLIDriver`, and thus, please update the function description of `newTemporaryConfiguration`.